ertviz/controllers/ensemble_selector_controller.py

```diff
@@ -0,0 +1,88 @@
+import json
+import random
+from dash.dependencies import Input, Output, State
+from dash.exceptions import PreventUpdate
+from ertviz.data_loader import get_ensembles
+from ertviz.models import EnsembleModel
+
+
+def ensemble_selector_controller(parent, app):
+    @app.callback(
+        Output(parent.uuid("ensemble-selector"), "layout"),
+        [Input("dropdown-layout", "value")],
+    )
+    def update_cytoscape_layout(layout):
+        return {"name": layout}
+
+    @app.callback(
+        Output(parent.uuid("ensemble-selector"), "elements"), [Input("url", "pathname")]
+    )
+    def update_ensemble_selector(query):
```

Comment:
where's the query param (url) actually used?

Reply:
it is not actually used, but url input is used as a trigger to update the graph. Might be a better way for this..